pycqed/analysis_v2/residual_ZZ_analysis.py

```diff
@@ -0,0 +1,192 @@
+import pycqed.analysis_v2.base_analysis as ba
+from pycqed.analysis.analysis_toolbox import get_datafilepath_from_timestamp
+import os
+from pycqed.analysis import analysis_toolbox as a_tools
+import matplotlib.pyplot as plt
+import numpy as np
+import matplotlib.pylab as pl
+from matplotlib import cm
+import pycqed.measurement.hdf5_data as h5d
+import lmfit
+from pycqed.analysis import fitting_models as fit_mods
+from pycqed.analysis.tools.plotting import set_xlabel, set_ylabel, \
+    cmap_to_alpha, cmap_first_to_alpha
+
+class Residual_Crosstalk(ba.BaseDataAnalysis):
+    """
+    Residual crosstalk analysis.
+
+    Produces residual crosstalk matrix from multiple crosstalk measurements.
```

Comment:
What are crosstalk measurements? 
Crosstalk is a widely used but ill defined concept. One should explain what experiment it uses either through a reference or by explaining it here.